client/src/mvc/tool/tool-form.js

```diff
@@ -49,6 +50,25 @@ const View = Backbone.View.extend({
                                     window.location = getAppRoot();
                                     return;
                                 }
+
+                                const client_version = options.version || options.tool_id;
+
+                                if (
+                                    data.versions &&
+                                    client_version &&
+                                    !data.versions.includes(client_version.replace(/.*\//, ""))
```

Comment:
is this valid syntax ? wouldn't this need to be
```suggestion
                                    !data.versions.includes(client_version.replace("/.*\//", ""))
```

Reply:
I also don't think we'd want to display this everywhere that we load a tool model. This should probably be a warning on the tool form, which is the only place where a user could do anything about this. If multiple tool forms are open this is also going to trigger toasts all over the place. I don't know how exactly we're handling this in backbone, but in Vue you'd emit an event and then bind an error handler to the event that renders a warning in the correct place.

Reply:
It's valid, javascript regexes can be defined inline like that.

Reply:
Whoops, didn't see the other comment when I responded, only the bit about the syntax.  I agree with @mvdbeek about how this should be displayed; toasts aren't a great fit here.